app/api/dds/v1/project_transfers_api.rb

```diff
@@ -56,7 +56,23 @@ class ProjectTransfersAPI < Grape::API
         authenticate!
         project = Project.find(params[:project_id])
         policy_scope(ProjectTransfer).where(project: project)
-     end
+      end
+
+      desc 'View project transfer' do
+        detail 'Used to retrieve the metadata template instance for a corresponding DDS object.'
+        named 'view object metadata'
+        failure [
+          [200, 'Success'],
+          [401, 'Unauthorized'],
+          [404, 'Project does not exist']
```

Comment:
The 404 description should be 'Project transfer does not exist'

Also, since this is an authorized resource, 403 'Forbidden' should be part of the failure response list.